File: even-odd/evenOrOdd.js
var stdin = process.openStdin();

console.info("Please enter a number: ")
stdin.addListener("data", function(d) {
    evenOrOdd(d)
    console.info("Please enter another number: ")
  });

function evenOrOdd(input) {
    if (isEven(input)) {
        console.info("The number you entered is even");
    } else {
        console.info("The number you entered is odd");
    }
}

function isEven(input) {
    return input % 2 === 0;
}

File: even-odd/evenOrOdd.go
package main

import "fmt"

func main() {
	fmt.Print("Enter a number: ")
	var input int
	fmt.Scan(&input)

	if isEven(input) {
		fmt.Print("The number you entered is even")
	} else {
		fmt.Print("The number you entered is odd")
	}
}

func isEven(number int) bool {
	return number%2 == 0
}
